src/installing-workflow/system-requirements.md

```diff
@@ -43,3 +43,10 @@ Any Kubernetes 1.1 cluster should also use a Docker version < 1.10.0 so that `ku
 ## Docker Insecure Registry
 
 The on-cluster, Deis-managed Docker registry is not deployed with TLS by default. As such, all Kubernetes worker nodes must have their Docker daemons configured to use an insecure registry. The subnet should encompase any private networks used by your worker nodes including any overlay networks. Depending on your Kubernetes and Docker configuration, setting `EXTRA_DOCKER_OPTS="--insecure-registry=10.0.0.0/8"` may be sufficient.
+
+## Selinux + Overlayfs
+
+If you are using Docker with Overlayfs you must disable selinux by adding `--selinux-enabled=false` to `EXTRA_DOCKER_OPTS`. For more background information see:
```

Comment:
If we are capitalizing Overlayfs we should use the proper form: OverlayFS

Also to remain consistent, we should use SELinux for proper capitalization here as well